oauth/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "librespot-oauth"
+version = "0.5.0-dev"
+rust-version = "1.73"
+authors = ["Paul Lietar <paul@lietar.net>"]
```

Comment:
I think you should change it to your name. After all, you wrote this part, not Paul.